src/ImageSharp/Image.LoadPixelData.cs

```diff
@@ -33,7 +33,7 @@ public static Image<TPixel> LoadPixelData<TPixel>(TPixel[] data, int width, int
         /// <param name="height">The height of the final image.</param>
         /// <typeparam name="TPixel">The pixel format.</typeparam>
         /// <returns>A new <see cref="Image{TPixel}"/>.</returns>
-        private static Image<TPixel> LoadPixelData<TPixel>(Span<TPixel> data, int width, int height)
+        public static Image<TPixel> LoadPixelData<TPixel>(Span<TPixel> data, int width, int height)
```

Comment:
We should use `ReadOnlySpan` in all overloads.

Reply:
My mistake that I also forgot about this in #565 :)

Reply:
Agreed. I'll update the PR shortly. Also need to figure out why that test is failing.

Reply:
Updated.